cvat-ui/src/components/annotation-page/standard-workspace/objects-side-bar/label-item.tsx

```diff
@@ -48,22 +62,37 @@ function LabelItemComponent(props: Props): JSX.Element {
 
     return (
         <Row
-            align='middle'
+            align='stretch'
             justify='space-around'
-            className='cvat-objects-sidebar-label-item'
-            style={{ display: visible ? 'flex' : 'none' }}
+            className={[
+                'cvat-objects-sidebar-label-item',
+                visible ? '' : 'cvat-objects-sidebar-label-item-disabled',
+            ].join(' ')}
         >
-            <Col span={4}>
-                <Button style={{ background: labelColor }} className='cvat-label-item-color-button'>
+            <Col span={2}>
+                <div style={{ background: labelColor }} className='cvat-label-item-color'>
                     {' '}
-                </Button>
+                </div>
             </Col>
-            <Col span={14}>
-                <Text strong className='cvat-text'>
-                    {labelName}
-                </Text>
+            <Col span={11}>
+                <CVATTooltip title={labelName}>
+                    <Text strong className='cvat-text'>
+                        {labelName}
+                    </Text>
+                </CVATTooltip>
             </Col>
             <Col span={3}>
+                <LabelKeySelectorPopover
+                    keyToLabelMapping={keyToLabelMapping}
+                    labelID={labelID}
+                    updateLabelShortcutKey={updateLabelShortcutKey}
+                >
+                    <Button className='cvat-label-item-setup-shortcut-button' size='small' ghost type='dashed'>
+                        {labelShortcutKey}
+                    </Button>
+                </LabelKeySelectorPopover>
+            </Col>
+            <Col span={3} offset={1}>
```

Comment:
I guess we can enlarge space for label name by making span equal to 2. It will look like this.

![изображение](https://user-images.githubusercontent.com/8168727/113826240-9968df00-978a-11eb-9dd9-0ec7113be455.png)

Reply:
Done